frontend/src/pages/Methodology/methodologyComponents/GlossaryTerm.tsx

```diff
@@ -20,7 +20,7 @@ export const parseDescription = (description: string) => {
   const elements: any[] = []
   let remainingText = description
 
-  while (remainingText.length > 0) {
+  while (remainingText?.length > 0) {
```

Comment:
i added this in; I think we'll eventually factor out this section but for now an empty `remainingText` was throwing an error because it wont have a length. adding the optional chaining `?.` allows that error to instead just be `undefined`